src/discounts/views/VoucherDetails/VoucherDetails.tsx

```diff
@@ -224,6 +224,7 @@ export const VoucherDetails: React.FC<VoucherDetailsProps> = ({
           notifySaved();
           closeModal();
           reset();
+          refetch();
```

Comment:
side note: I think we have to start thinking about how to manage Apollo cache properly to avoid manual refetches